src/Components/MyRockets/RocketItem.js

```diff
@@ -0,0 +1,52 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { useDispatch } from 'react-redux';
+import { ReserveRocket } from '../../redux/rockets/Rocket';
+
+const RocketItem = (props) => {
+  const {
+    id, name, description, flickrImage, reserved,
+  } = props;
+
+  const dispatch = useDispatch();
+
+  const onClick = () => {
+    dispatch(ReserveRocket(id));
+  };
+
+  return (
+    <div>
+      <li className="rocketitem" id="flex">
+        <img src={flickrImage} alt="Girl in a jacket" width="250" height="250" />
+        <div className="rocketright" id="flex">
+          <h2>{name}</h2>
+          <div className="spantext" id="flex">
+            <span className={reserved ? 'active' : 'inactive'}>
+              reserved
+            </span>
```

Comment:
- [ ] According to the requirements, you should use React's conditional rendering syntax:

![image](https://user-images.githubusercontent.com/77069011/217966570-205c90df-6ffb-413c-9297-4abbd938fa63.png)

Conditional rendering and toggling the visibility of the elements using CSS are different! Please consider applying this change. 🤓 🔧